script/legacy/2019-05-14-armagaddon/delete-from-gcs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+## Affected period to delete is
+## 2019-05-04T11:00:00Z to 2019-05-11T11:00:00Z
+
+## Deletion in these buckets requires ops credentials.
+
+function delete_hours() {
+    local target_dir=$1
+
+    gsutil -m rm -r $target_dir/2019-05-04/11/
+    gsutil -m rm -r $target_dir/2019-05-04/12/
+    gsutil -m rm -r $target_dir/2019-05-04/13/
+    gsutil -m rm -r $target_dir/2019-05-04/14/
+    gsutil -m rm -r $target_dir/2019-05-04/15/
+    gsutil -m rm -r $target_dir/2019-05-04/16/
+    gsutil -m rm -r $target_dir/2019-05-04/17/
+    gsutil -m rm -r $target_dir/2019-05-04/18/
+    gsutil -m rm -r $target_dir/2019-05-04/19/
+    gsutil -m rm -r $target_dir/2019-05-04/20/
+    gsutil -m rm -r $target_dir/2019-05-04/21/
+    gsutil -m rm -r $target_dir/2019-05-04/22/
+    gsutil -m rm -r $target_dir/2019-05-04/23/
+
+    for day in $(seq -f "%02g" 05 10); do
```

Comment:
seq has a flag for fixed width:
```suggestion
    for day in $(seq -w 5 10); do
```

Reply:
What I want is exactly 2 digits zero-padded, so I'm going to avoid `-w` out of paranoia, but it's good to know about.